nodejs/test/integration/serverless.yml

```diff
@@ -0,0 +1,226 @@
+service: cjs
+frameworkVersion: '3'
+
+custom:
+  useESM:
+    default: false
+    cjs: false
+    esm: true
+
+plugins:
+  - serverless-offline
+
+provider:
+  name: aws
+  runtime: nodejs18.x
```

Comment:
While it's potentially complicating things, I'd wonder what results we'd get under other runtimes as well. We've encountered some language differences between runtimes before.

Which takes precedence when we're running in CircleCI jobs? The CircleCI environment or the `serverless-offline` provider runtime?

Reply:
(We could probably just update this line to use an env var from the CircleCI job, to make sure.)